### ISSUE TITLE
scandir unable to find a working combination

### ISSUE THREAD

**Issue by asratgh** [hasse69/rar2fs#109]
I am using armbian on orangepizero board.
``` 
Linux op0dl 4.19.20-sunxi #5.75 SMP Sat Feb 9 19:02:47 CET 2019 armv7l GNU/Linux
``` 
I tried 2 patches also, but it is giving same error. I tried to compile 1.27.1 i.e latest version (via tar.gz, zip, git clone). Everytime I get the same error.

**Comment by hasse69** [comment]
Can you please attach your `config.log` file here.

**Comment by hasse69** [comment]
Also, if possible, please attach your `/usr/include/dirent.h`.

**Comment by hasse69** [comment]
And while you are at it...
```
gcc -dM -E - < /dev/null | grep LARGEFILE
```

**Comment by hasse69** [comment]
Any news here? I would really like to look into this issue before going for a new release of rar2fs.

**Comment by asratgh** [comment]
> Any news here? I would really like to look into this issue before going for a new release of rar2fs.

Hi, today I reinstalled Armbian 5.75 orangepizero. I did apt update/upgrade. Followed git install method in wiki section. Now it successfully compiled. No errors. May be my microsd card was corrupt, that's why I was getting errors. I have also attached config.log and dirent.h of this successful build.  I dont have Files/logs of previous build. ... ... You have made a very useful software, thanks thanks. 
[config.log](https://github.com/hasse69/rar2fs/files/2972594/config.log)
[config-dirent.zip](https://github.com/hasse69/rar2fs/files/2972618/config-dirent.zip)
